src/Iact1dUnbinnedLkl.cc

```diff
@@ -932,26 +932,17 @@ Int_t Iact1dUnbinnedLkl::ReaddNdESignal(Int_t nFiles, TString* filenames, Float_
         }
      }
 
-  // perform the linear combination
   if(status==0)
     {
-      Double_t linC = 0;
-      for(Int_t ibin=0;ibin<nBins;ibin++)
+      // create histogram for the linear combination of dNdESignal
+      linCHdNdESignal = new TH1F("linCHdNdESignal","dN/dE for signal events",nBins,lowEdge,highEdge);
```

Comment:
No need to define title, or axis titles, because this is just a provisional histogram and that information will never be used.

Reply:
Done!